apps/marketplace/main.scss

```diff
@@ -26,6 +26,10 @@
   }
 }
 
+.marginTop0 {
+  margin-top: 0rem
```

Comment:
Once Mar 3481 is merged this will make sense.

Reply:
Should these changes be added with the pull request for MAR-3481?